.drone.yml

```diff
@@ -0,0 +1,9 @@
+pipeline:
+  eslint:
+    image: nextcloudci/eslint
```

Comment:
There is no `latest` tag for the image. Replace it with `nextcloudci/eslint:eslint-1`(the git tag is named like this and I added this as automated build).

Also that then allows to fetch a new version of the image, otherwise you need to do a docker pull right on the drone server, which doesn't scale (because only Lukas and I can do this) ;)